test/unit/spec/LoginRouter_spec.js

```diff
@@ -1400,6 +1405,7 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, Logger, OktaAut
             test.router.enrollCall();
             return Expect.waitForEnrollCall(test);
           })
+          .then(tick)
```

Comment:
do we need this ```tick```? we are already waiting for an enroll call. What happens here that we need this?

Reply:
This tick it to give Chosen time to update its UI elements.

Reply:
should this then be something like your ```waitForCountryCodeSelect```?

Reply:
In that test, we check that the value of the select was updated.

Only thing I can `waitFor` is the specific value for the `this.inputWrap(field).find('.chzn-single span').trimmedText()`, which is the same as we expect in the`expect(test.form.selectedCountry()).toBe(countryText);`.

Should I replace the `expect(...selectedCountry()).toBe(countryText)` with the `waitFor` logic?

Reply:
ok, it is fine then, just add a comment please
```.then(tick) // Wait for chosen items to render```